functions/expression/1.3/index.js

```diff
@@ -0,0 +1,23 @@
+import templayed from "../../utils/templayed";
+import { variableMap } from "../../utils/utilityFuncs";
+
+const customExpression = async ({ expression, variables }) => {
+    const parsedVars = variableMap(variables);
+    const template = templayed(expression)(parsedVars);
+    let functionOutput;
+
+    try {
+        functionOutput = new Function(`return ${template}`)();
+    }
+    catch (error) {
+        const errorMessage = `Error evaluating expression: "${error.message}" (template: ${template} variables: ${JSON.stringify(parsedVars)})`;
+        console.error(errorMessage);
```

Comment:
I think it is better to put this behind an advanced toggle (for instance 'debug') instead of always console logging this error.

Reply:
I could include it, but I was also thinking that when the "Advanced logging" is not enabled it will only return "Unexpected token" which is also pretty vague. Or is it regarding code being displayed on the page-builder on error?

Reply:
After discussing it with a colleague we think it's also a good idea. I've committed the suggestion and also fixed the indentation with prettier.